pydov/types/interpretaties.py

```diff
@@ -344,3 +344,92 @@ class LithologischeBeschrijvingen(AbstractBoringInterpretatie):
         'sourcefield': 'Y_mL72',
         'type': 'float'
     }]
+
+
+class GecodeerdeLithologieLaag(AbstractDovSubType):
+
+    _name = 'gecodeerde_lithologie_laag'
+    _rootpath = './/gecodeerdelithologie/laag'
+
+    _fields = [{
+        'name': 'diepte_laag_van',
+        'source': 'xml',
+        'sourcefield': '/van',
+        'definition': 'Diepte van de bovenkant van de laag lithologische'
+                      ' beschrijving in meter.',
+        'type': 'float',
+        'notnull': False
+    }, {
+        'name': 'diepte_laag_tot',
+        'source': 'xml',
+        'sourcefield': '/tot',
+        'definition': 'Diepte van de onderkant van de laag lithologische'
+                      ' beschrijving in meter.',
+        'type': 'float',
+        'notnull': False
+    }, {
+        'name': 'hoofd_grondsoort',
+        'source': 'xml',
+        'sourcefield': '/hoofdnaam/grondsoort',
+        'definition': 'grondsoort (als code) van de laag gecodeerde '
+                      'lithologie',
+        'type': 'string',
+        'notnull': False
+    }, {
+        'name': 'bijmenging_plaatselijk',
+        'source': 'xml',
+        'sourcefield': '/bijmenging/plaatselijk',
+        'definition': 'plaatselijk of niet-plaatselijk',
+        'type': 'string',
+        'notnull': False
+    }, {
+        'name': 'bijmenging_hoeveelheid',
+        'source': 'xml',
+        'sourcefield': '/bijmenging/hoeveelheid',
+        'definition': 'aanduiding van de hoeveelheid bijmenging',
+        'type': 'string',
+        'notnull': False
+    }, {
+        'name': 'bijmenging_grondsoort',
+        'source': 'xml',
+        'sourcefield': '/bijmenging/grondsoort',
+        'definition': 'type grondsoort (als code) van de laag '
+                      'gecodeerde lithologie of geotechnische '
+                      'codering',
```

Comment:
Maybe add something from the XSD definition of 'bijmenging': "lithologisch nevenbestanddeel (als code) van de laag gecodeerde lithologie"